documentation/staging/content/faq/external-clients.md

```diff
@@ -112,23 +141,50 @@ is hosted outside of the Kubernetes cluster, then:
   For example, `http://my-lb-address:my-lb-port`.
 
 - You may need to [enable unknown host access](#enabling-unknown-host-access)
-  on the WebLogic Servers that host the EJB or JMS resources.
+  on the WebLogic Server instances that host the EJB or JMS resources.
+
+- If tunneling, then you may need to configure clusters that host EJB or
+  JMS resources with a 'server affinity' default load balancer algorithm.
+  This can significantly speedup connection creation for EJB and JMS clients.
+  See [Configuring WebLogic Server affinity load balancing algorithms](#configuring-weblogic-server-affinity-load-balancing-algorithms).
+
+- __JTA NOTE__: The operator does not currently support external WebLogic JTA access
```

Comment:
I do not understand this note.  Once Anthony's change of RJVM Forwarding the fact that every managed server has the same port number should not be a cross domain communication limitation.  Each server has a different service name which makes the CoordinatorURL or SubCoordinatorURL unique even when the ports are the same.    Is this not the case?  Please see Alex's blog post https://blogs.oracle.com/weblogicserver/weblogic-server-jta-in-a-kubernetes-environment

The original limitation of cross domain transaction communication between domains where one is inside of K8s and one is outside of k8s is due to load balancing.  With Anthony's change RJVM Forwarding any 2PC calls will always end up being processed by the same Managed Server.

Reply:
Removed the JTA Notes as RJVM Forwarding documentation will address these JTA limitations.